syft/frameworks/crypten/hook/hook.py

```diff
@@ -12,10 +13,10 @@ def get_hooked_crypten_func(func_api_name, func):
     @tracer(func_name=cmd_name)
     @wraps(func)
     def overloaded_func(*args, **kwargs):
-        try:
+        if syft.hook.trace.active:
```

Comment:
Q: I needed to check if the tracer was or not active (tracer is active if we build a plan)